0128-energy-subnetwork.md

```diff
@@ -191,24 +191,25 @@ Incentivize an evenly distributed and continuous data connection; which tells a
+| 1     | Read DER Data                                    | 0.01       |
+| 2     | Read Validated DER Data                          | 1          |
+| 3     | Read Validated DER Data + Control Qualified DERs | 2          |
 
 ##### Validation and Qualification Mechanism
 
 Validation and Qualification is offered by the Service Provider to DERs that want to participate in Energy Services. Validation is achieved by the DER Host staking $10 USD worth of ENERGY tokens per kW available power for 45 days with a 10 day un-staking period. During staking the DER is subject to additional Service Provider audits and tests. The DER Host must comply with requirements and provide needed information promptly. Failure to comply with the requirements the staked ENERGY tokens will be forfeited and burned. A validated and controllable DER can be Qualified by the Service Provider. A Qualified DER can be opted in to be available for energy services by the DER Host.
 
 #### Calculation
 
-Each DER datapoint gets a score based on the sum of the baseline reward principles for a given reward period, e.g. 1h. The scores are summarized for a given period. When the sum of scores for each DER is calculated, each DER is rewarded ENERGY tokens based on its fraction of the total score and the token distribution level for that period.
+Each DER datapoint gets a score based on the sum of principles one and two multipled by three for a given reward period, e.g. 1h. The scores are summarized for a given period. When the sum of scores for each DER is calculated, each DER is rewarded ENERGY tokens based on its fraction of the total score and the token distribution level for that period.
 
-Example: A DER has been connected to the network for 5 weeks (Uptime Level 1). During the reward epoch of one hour the DER has provided 500 data points, but it had some connection problem for two minutes so there is no data for two consecutive minutes (Quality Level 3). The DER has not been validated (Value Level 1). This gives a score of 500 x (100 + 60 + 100) = 130 000
+Example: A DER has been connected to the network for 5 weeks (Uptime Level 1). During the reward epoch of one hour the DER has provided 500 data points, but it had some connection problem for two minutes so there is no data for two consecutive minutes (Quality Level 3). The DER has not been validated (Value Level 1). This gives a score of 500 x (100 + 60) 0.01 = 800
 
-The total scores for the hour is the sum of all DER scores, e.g. 2&nbsp;000&nbsp;000 and the DER’s part of this is then 130&nbsp;000&nbsp;/&nbsp;2&nbsp;000&nbsp;000&nbsp;=&nbsp;0.065. The amount of ENERGY tokens issued to the DER is then 6.5% of the DER Host rewards emissions.
+The total scores for the hour is the sum of all DER scores, e.g. 2&nbsp;000&nbsp;000 and the DER’s part of this is then 800 /&nbsp;2&nbsp;000&nbsp;000&nbsp;=&nbsp;0.0004. The amount of ENERGY tokens issued to the DER is then 0.04% of the DER Host baseline rewards emissions.
```

Comment:
```suggestion
The total scores for the hour is the sum of all DER scores, e.g. 2&nbsp;000&nbsp;000 and the DER’s part of this is then $\frac{800}{2\ 000\ 000} = 0.0004$. The amount of ENERGY tokens issued to the DER is then 0.04% of the DER Host baseline rewards emissions.
```

perhaps this? $\frac{800}{2\ 000\ 000} = 0.0004$
I think it should work inline.